test/e2e/config/conf.js

```diff
@@ -36,13 +47,21 @@ const config = {
   framework: 'custom',
   frameworkPath: require.resolve('protractor-cucumber-framework'),
   specs: ['../features/**/*.feature'],
-  baseUrl: process.env.TEST_URL || 'http://localhost:2500',
+  // specs: [
```

Comment:
please remove or comment why is commented out

Reply:
feature improvements